src/components/wallet/views/loggedInContent.tsx

```diff
@@ -55,5 +55,17 @@ export default function loggedInContent(this: Wallet) {
         </pre>
       </collapsible-container>
     ) : null,
+
+    loHas(this.account, 'state') ? (
```

Comment:
Can you explain what `loHas` and `loOmit` are? 

Based on what I'm understanding, I think this line needs to be changed to:
```suggestion
    loHas(this.account, 'claimableBalances') ? (
```

Otherwise the component will be rendered whenever `state` changes, not `claimableBalances` right?

Reply:
Good catch!